Tests/TuistLoaderTests/Up/UpRomeTests.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistSupport
+import XCTest
+
+@testable import TuistLoader
+@testable import TuistSupportTesting
+
+final class UpRomeTests: TuistUnitTestCase {
+    var platforms: [Platform]!
+    var cachePrefix: String!
+    var upHomebrew: MockUp!
+    var rome: MockRome!
+    var subject: UpRome!
+
+    override func setUp() {
+        super.setUp()
+        platforms = [.iOS]
+        cachePrefix = "Swift_1_2"
```

Comment:
Maybe we could also add tests for an empty / nil `platforms` param and no `cache prefix`? 🤔